### PR TITLE
docs(gpu): understanding nvidia fp8 format

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: compute/gpu/reference-content/understanding-nvidia-fp8.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+dates:
+  validation: 2023-10-23
+  posted: 2023-10-23
+categories:
+  - compute
+---
+
+With the release of the H100 GPU NVIDIA introduced support for a new datatype, FP8 (8-bit floating point), enabling higher throughput of matrix multiplies and convolutions.
+
+FP8, an 8-bit floating point standard, was jointly developed by NVIDIA, ARM, and Intel to speed up AI development by improving memory efficiency during AI training and inference processes.
```
Comment (nerda-codes) [inline compute/gpu/reference-content/understanding-nvidia-fp8.mdx:18]:
```suggestion
FP8 is an 8-bit floating point standard which was jointly developed by NVIDIA, ARM, and Intel to speed up AI development by improving memory efficiency during AI training and inference processes.
```

File: compute/gpu/reference-content/understanding-nvidia-fp8.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+The ongoing industry evolution is evident in the shift from 32-bit to 16-bit, and currently, to 8-bit precision formats. This change is especially advantageous for transformer networks, crucial AI breakthroughs, as they perform better with the detailed precision provided by 8-bit floating point formats.
+
+This uniform standard ensures compatibility across diverse hardware and software environments, thus driving computing capabilities to unprecedented levels.
+
+FP8 sets forth two distinct eight-bit floating-point formats, E5M2 and E4M3, to facilitate interoperability between different hardware platforms. It aligns closely with the current IEEE 754 standards for floating-point computations, achieving an ideal harmony between hardware efficiency and software performance. This strategy seeks to leverage existing frameworks, quicken uptake, and boost developer efficiency.
```
Comment (nerda-codes) [inline compute/gpu/reference-content/understanding-nvidia-fp8.mdx:24]:
```suggestion
FP8 sets forth two distinct eight-bit floating-point formats, E5M2 and E4M3, to facilitate interoperability between different hardware platforms. It aligns closely with the current IEEE 754 standards for floating-point computations, achieving an ideal harmony between hardware efficiency and software performance. This strategy seeks to leverage existing frameworks, speed up uptake, and boost developer efficiency.
```

**Review by nerda-codes — APPROVED** [review]

File: compute/gpu/reference-content/understanding-nvidia-fp8.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  paragraph: This section provides information about NVIDIA FP8 (8-bit floating point) format
+tags: NVIDIA FP8 GPU cloud
+dates:
+  validation: 2023-10-23
+  posted: 2023-10-23
+categories:
+  - compute
+---
+
+With the release of the H100 GPU, NVIDIA introduced support for a new datatype called FP8 (8-bit floating point), enabling higher throughput of matrix multiplies and convolutions.
```
Comment (RoRoJ) [inline compute/gpu/reference-content/understanding-nvidia-fp8.mdx:16]:
```suggestion
With the release of the H100 GPU, NVIDIA introduced support for a new datatype called FP8 (8-bit floating point), enabling higher throughput of matrix multipliers and convolutions.
```

Comment (RoRoJ) [reply compute/gpu/reference-content/understanding-nvidia-fp8.mdx:16]: Not sure if this should be multipliers or multiples?

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**